src/tools/rbd_mirror/group_replayer/Replayer.cc

```diff
@@ -1089,9 +1136,22 @@ void Replayer<I>::handle_remove_mirror_peer_uuid(
     int r, const std::string &snap_id) {
   dout(10) << snap_id << ", r=" << r << dendl;
 
-  if (r < 0) {
+  // ENOENT could be resulting because mirror_peer_uuid was removed before,
```

Comment:
Misleading comment - the call that failed would be group_snap_set - it does not know about the peer_uuids in the snapshot struct.